tests/qos/cpu_qos_test.cc

```diff
@@ -505,6 +519,274 @@ TEST_P(CpuQosTestWithoutIxia, PerEntryAclCounterIncrementsWhenEntryIsHit) {
+                           }
+                         }
+                         payload: "ARP broadcast packet."
+                       )pb"));
+
+  // LLDP multicast packet.
+  // TODO: If packetlib starts supporting LLDP, we can replace this
+  // LLDP packet hex dump with a readable protobuf. For now, we can verify that
+  // this is indeed a valid LLDP packet using, e.g., https://hpd.gasmi.net/.
+  static constexpr absl::string_view kLldpPacketHexDump =
+      "0180c200000ef40304321f6688cc02070402320af046030404073235330602007808266a"
+      "753166326d3168342e6d747631352e6e65742e676f6f676c652e636f6d3a62702d342f36"
+      "31100c05010af0460302000000fd000a1e6a753166326d3168342e6d747631352e6e6574"
+      "2e676f6f676c652e636f6dfe0c001a11041666534220c811b3fe05001a1105920000";
+  packetlib::Packet packet =
+      packetlib::ParsePacket(absl::HexStringToBytes(kLldpPacketHexDump));
+  if (packet.headers_size() < 1 || !packet.headers(0).has_ethernet_header()) {
+    return gutil::InternalErrorBuilder();
+  }
+  packet.mutable_headers(0)
+      ->mutable_ethernet_header()
+      ->set_ethernet_destination("01:80:C2:00:00:0E");  // LLDP multicast.
+  packets.push_back(packet);
+
+  // Post-process packets to ensure they are valid.
+  for (auto &packet : packets) {
+    RETURN_IF_ERROR(packetlib::PadPacketToMinimumSize(packet).status());
+    RETURN_IF_ERROR(packetlib::UpdateAllComputedFields(packet).status());
+    LOG(INFO) << packet.DebugString();  // TODO: remove.
+  }
+  return packets;
+}
+
+// Queries via gNMI, parses, and returns as a proto the gNMI path
+// `qos/interfaces/interface[interface-id=CPU]/output/queues`, which contains
+// the state of all CPU queue counters.
+absl::StatusOr<openconfig::QueuesByName> GetCpuQueueStateViaGnmi(
+    gnmi::gNMI::StubInterface &gnmi_stub) {
+  ASSIGN_OR_RETURN(
+      std::string queues_json,
+      GetGnmiStatePathInfo(
+          &gnmi_stub,
+          "qos/interfaces/interface[interface-id=CPU]/output/queues",
+          "openconfig-qos:queues"));
+
+  google::protobuf::util::JsonParseOptions options;
+  options.ignore_unknown_fields = true;
+  openconfig::Queues queues_proto;
+  RETURN_IF_ERROR(
+      gutil::ToAbslStatus(google::protobuf::util::JsonStringToMessage(
+          queues_json, &queues_proto, options)));
+
+  // Convert `Queues` to `QueuesByName`, which is equivalent but more convenient
+  // for diffing.
+  openconfig::QueuesByName queues_by_name;
+  for (auto &queue : queues_proto.queues()) {
+    queues_by_name.mutable_queues()->insert({queue.name(), queue.state()});
+  }
+
+  return queues_by_name;
+}
+
+// Purpose: Verify that the CPU is protected from packets by default.
+TEST_P(CpuQosTestWithoutIxia,
+       NoUnexpectedPacketsReachCpuInPristineSwitchState) {
+  LOG(INFO) << "-- START OF TEST ---------------------------------------------";
+  Testbed().Environment().SetTestCaseID("ea20865e-e15a-4d21-947c-6f638937fbec");
```

Comment:
No TestCaseID should be in any PR, can you pls fix this ?

Reply:
Done.